src/bin/tupm.rs

```diff
@@ -252,15 +252,15 @@ fn main() {
         }));
 
     // Determine the database password, if possible
-    let password = if matches.is_present("password") {
+    let password = if matches.is_present("test") {
```

Comment:
This change doesn't need to be present, as far as I know. We don't need to always prompt the user for a password before entering the screen-oriented UI.

Reply:
Yeah, that shouldn't be in here. Not sure why my repo is so different from yours, but I will clean it up.